addons/website/views/snippets/s_references.xml

```diff
@@ -2,27 +2,28 @@
 <odoo>
 
 <template id="s_references" name="References">
-    <section class="s_references pt48 pb32">
-        <div class="container">
-            <h2 style="text-align: center;">Our References</h2>
-            <p style="text-align: center;">We are in good company.</p>
-            <div class="row">
-                <div class="col-lg-2 pt16 pb16">
+    <section class="s_references o_cc o_cc1 pt80 pb80">
+        <div class="container" style="text-align: center;">
+            <h2>Partners and references</h2>
+            <p class="lead">Use this section to boost your company's credibility.</p>
+            <a href="#">See our case studies <i class="fa fa-long-arrow-right ms-2"/></a>
+            <div class="row align-items-center justify-content-center mt-4">
```

Comment:
> Remove the align-items-center and justify-content-center classes: firstly because there is no option to add/remove them, and secondly because they don't seem to have a meaningful effect (nothing changed when I remove them) 🙂

I do agree that this enforces a class for the user and we shouldn't do this, but these classes are helpful to ensure a good horizontal and vertical alignment between all the logos.

In my opinion it's fine to take some liberty if it provides a better visual result, but maybe you have a different opinion, feel free to tell me what you think about it 👍 

| | With | Without |
|--------|--------|--------|
|`align-items-center` | <img alt="image" src="https://github.com/user-attachments/assets/fdce24f1-76d0-40fa-8a68-6d3b33352443"> | <img alt="image" src="https://github.com/user-attachments/assets/62a7f2de-1bf6-4eee-bbc6-d002246ed8b0"> |
|`justify-content-center`  | <img alt="image" src="https://github.com/user-attachments/assets/fd1ede2b-d7f9-4777-b0a5-1096747fe8d5"> | <img alt="image" src="https://github.com/user-attachments/assets/ad058632-1aef-4321-8c79-f32f43da3d65"> |
| both classes | <img alt="image" src="https://github.com/user-attachments/assets/a2a4b4eb-1fca-4f50-a845-02f99e8e99c8"> | <img alt="image" src="https://github.com/user-attachments/assets/64c63d84-03a1-4b41-ab5a-9ea59453befd"> |

Reply:
For me, we should still not add these classes, because there are other ways to achieve the same visual :slightly_smiling_face: :
- `align-items`: we can use the padding to make the images aligned as we want (like if we add padding to the smaller blocks, they can be centered) :slightly_smiling_face: (Moreover, given this snippet, I think the user should try to upload images of similar sizes anyway, but if not, he can use the padding)
- `justify-content`: we can resize the first column from the left, in order to add an offset (it will make the column smaller, but then we can resize it from the right to increase its size) => this will allow to center the items :wink: (this is what we do when setting 5 columns with the "Column Count" option)

Note that if you are not convinced by my `align-items` arguments, we have a "Vertical Alignment" option that does what you want, but it is only available for some snippets (Text-Image, Image-Text and Columns) :arrow_down: 
```xml
 <!--  Vertical Alignment -->
    <div data-option-name="vAlignment" id="row_valign_snippet_option" data-selector=".s_text_image, .s_image_text, .s_three_columns" data-target=".row">
        <we-button-group class="o_we_sublevel_1" string="Vert. Alignment" title="Vertical Alignment" data-dependencies="normal_mode">
            <we-button title="Align Top"
                       data-select-class="align-items-start"
                       data-img="/website/static/src/img/snippets_options/align_top.svg"/>
            <we-button title="Align Middle"
                       data-select-class="align-items-center"
                       data-img="/website/static/src/img/snippets_options/align_middle.svg"/>
            <we-button title="Align Bottom"
                       data-select-class="align-items-end"
                       data-img="/website/static/src/img/snippets_options/align_bottom.svg"/>
            <we-button title="Stretch to Equal Height"
                       data-select-class="align-items-stretch"
                       data-img="/website/static/src/img/snippets_options/align_stretch.svg"/>
        </we-button-group>
    </div>
```

If you really think that changing the padding or having images with similar sizes are not enough, you could add the `.s_references` snippet in the `data-selector` :slightly_smiling_face: 
But given that the new design is not really different from before, and since nobody complained, it may not be worth allowing this option for it now :slightly_smiling_face: